src/index.js

```diff
@@ -29,6 +29,17 @@ function render(ui, {container, baseElement = container, queries} = {}) {
       // Intentionally do not return anything to avoid unnecessarily complicating the API.
       // folks can use all the same utilities we return in the first place that are bound to the container
     },
+    asFragment: () => {
+      if (typeof document.createRange === 'function') {
```

Comment:
might wanna add a `/* istanbul ignore if */` here, and leave a comment pointing back to https://github.com/jsdom/jsdom/issues/317

Reply:
Thanks! I've just committed that 👍